.github/workflows/bsi-update.yml

```
@@ -0,0 +1,69 @@
+# This workflow runs "npm i" to see if any dependencies have updated.  If so, it opens a PR and approves it.
+
+name: BSI Update
+
+on:
+  workflow_dispatch: # manual trigger for testing
+
+jobs:
+  build:
+    # self-hosted doesn't work for public repos (security), so need to use ubuntu
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v2
+
+      - name: Use Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: '12.x'
+
+      - name: Fresh Dependency Install
+        run: npm install
+      
+      - name: Check For Changes
+        id: check-changes
+        run: |
+          echo "::set-output name=changes-exist::$(git diff --exit-code package-lock.json)"
```

```suggestion
          echo "::set-output name=changes-exist::$(git diff -s --exit-code package-lock.json && echo false || echo true)"
```

I think? Like `--exit-code` alone outputs the diff and sets the exit code to 1 if its non-empty... `-s` makes it not output the diff, but to get something usable I think you need to do the `&&`/`||` stuff.

If 0/1 work here (I don't remember) then this would work too:

```suggestion
          echo "::set-output name=changes-exist::$(git diff -s --exit-code package-lock.json ; echo $?)"
```

---

Yeah, this part was a guess until I could actually test things out.  I'll try one of these and see how it goes